priv/templates/phx.gen.auth/settings_live.ex

```diff
@@ -0,0 +1,152 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>SettingsLive do
+  use <%= inspect context.web_module %>, :live_view
+
+  alias <%= inspect context.module %>
+
+  on_mount {<%= inspect auth_module %>, :mount_current_<%= schema.singular %>}
+
+  def render(assigns) do
+    ~H"""
+    <h1>Settings</h1>
+
+    <h3>Change email</h3>
+
+    <.form
+      id="email_form"
+      :let={f}
+      for={@email_changeset}
+      phx-submit="update_email"
+      phx-change="validate_email"
+    >
+      <%%= if @email_changeset.action == :insert do %>
+        <div class="alert alert-danger">
+          <p>Oops, something went wrong! Please check the errors below.</p>
+        </div>
+      <%% end %>
+
+      <%%= label f, :email %>
+      <%%= email_input f, :email, required: true, value: input_value(f, :email) %>
```

Comment:
I wonder if we should throttle or debounce this validation? I'm seeing a database query each keystroke due to `Ecto.Changeset.unsafe_validate_unique/4`.

Reply:
I made some changes to validate the email only when the `email_changeset` changes are applied, just before writing to the database.